docs/DeveloperGuide.md

```diff
@@ -183,15 +182,15 @@ Apart from `Cooper`, the rest of the app consists of these seven components:
 #### Interaction of the architecture components to process user input
 - The sequence diagram below shows how cOOPer's components interact with each other when a user enters their **sign in details** at the _verification layer_.
 
-> ℹ️`userInput` represents the credentials input by the user for verification. For example, `register John pw 12345 as admin`.
+> ℹ️`userInput` represents the credentials input by the user for verification. For example, `register John /pw 12345 /as admin`.
 
 <p align="center">
     <img src="developerGuideDiagrams/signInSequenceDiagram.png" alt="signInSequenceDiagram"><br>
 </p> 
 
 - The next sequence diagram below shows how cOOPer's components interact with each other when a user enters a **command** at the _features layer_.
 
-> ℹ️ `userInput` represents a command input by the user. For example, `meetings`.<br>
+> ℹ️ `userInput` represents a command input by the user. For example, `meetings`.
```

Comment:
need the line break here or there will be formatting issues

Reply:
Weird it looks alright on typora. I will add them back then

Reply:
Hey I'll merge it first, and see how the website looks like